_posts/2024-01-10-15.0.0-release.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: post
+title: "Apache Arrow 15.0.0 Release"
+date: "2024-01-10 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 15.0.0 release. This covers
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 14.0.0 release, Curt Hagenlocher, Xuwei Fu, James Duong and Felipe Oliveira Carvalho
+have been invited to be committers.
+Jonathan Keane and Raúl Cumplido have joined the Project Management Committee (PMC).
+
+As per our tradition of rotating the PMC chair once a year
+Andy Grove was elected as the new PMC chair and VP.
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+## C Data Interface notes
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+### Compute layer
+
+### Datasets
+
+### Gandiva
+
+### Parquet
+
+### Substrait
+
+### Miscellaneous
+
+## C# notes
```

Comment:
@CurtHagenlocher could you help with the release notes?

Reply:
```suggestion
## C# notes

Removal of build targets:
 * Remove out-of-support versions of .NET and update C# README [GH-31579](https://github.com/apache/arrow/pull/39165)

New features:
 * Better support for decimal values which exceed the range of the BCL's System.Decimal [GH-38351](https://github.com/apache/arrow/pull/38481), [GH-38483](https://github.com/apache/arrow/pull/38508)
 * Expose ArrayDataConcentrator.Concatenate publicly [GH-38153](https://github.com/apache/arrow/pull/38154)
 * Add ToString methods to Arrow classes [GH-36566](https://github.com/apache/arrow/pull/30717)
 * Implement common interfaces for structure arrays and record batches [GH-38757](https://github.com/apache/arrow/pull/38759)
 * Make primitive arrays support IReadOnlyList<T?> [GH-38348](https://github.com/apache/arrow/pull/38680), [GH-39223](https://github.com/apache/arrow/pull/39224)
 * Add ToList to Decimal128Array and Decimal256Array [GH-37359](https://github.com/apache/arrow/pull/37383)
 * Support additional types Interval, Utf8View, BinaryView and ListView [GH-38316](https://github.com/apache/arrow/pull/39043), [GH-39341](https://github.com/apache/arrow/pull/39342)
 * Support creating FlightClient with Grpc.Core.Channel [GH-39335](https://github.com/apache/arrow/pull/39348)

Fixes and improved compatibility:
 * Make dictionaries in file and memory implementations work correctly and support integration tests [GH-32662](https://github.com/apache/arrow/pull/39146)
 * Support blank column names and enable more integration tests [GH-36588](https://github.com/apache/arrow/pull/39167)

```